src/api/chainHead_unstable_follow.md

```diff
@@ -26,6 +26,8 @@ This function works as follows:
 
 **Note**: This list of notifications makes it very easy for a JSON-RPC client to follow just the best block updates (listening to just `bestBlockChanged` events) or follow just the finalized block updates (listening to just `initialized` and `finalized` events). It is however not possible to easily figure out whether the runtime has been modified when these updates happen. This is not problematic, as anyone using the JSON-RPC interface naively propably doesn't need to account for runtime changes anyway.
 
+Additionally, the `chainHead_unstable_body`, `chainHead_unstable_call`, and `chainHead_unstable_storage` JSON-RPC function might cause the subscription to produce additional notifications.
```

Comment:
Should this maybe be more prominent and contained in the bullet list above? As a first time reader I think it is important to understand that the result notifications of these calls are generated on `chainHead_unstable_follow`.

Reply:
I think it's fine? I can't really find a good way to word this sentence if I put it in the bullet list above.